### ISSUE TITLE
Asset pipeline deployment

### ISSUE THREAD

**Issue by a-fro** [fourkitchens/emulsify#84]
Up until recently, I've been in the habit of using git hooks to add compiled code and build artifacts (CSS, etc) for deployment. Can you describe the approach you take at 4K for the assets that end up in the dist folder?

**Comment by a-fro** [comment]
Testing on Platform.sh, I followed [this suggestion](https://platform.sh/2016/11/platformsh-yarn) and used the [build hooks](https://docs.platform.sh/configuration/app/build.html) to run `gulp css`. Worked like a charm. 

Thoughts on deploying PatternLab to the same target for review?

**Comment by evanmwillhite** [comment]
This is kinda getting out of my wheelhouse, but I think we do something similar. On our past two projects on Acquia, we've used CircleCI to run the build processes and do testing (allows you to run npm tasks, etc.) and then we've just copied that build artifact over to Acquia's server (which doesn't allow you to run npm tasks). Ideally, you'd do the same build process during deployment - not sure what platform.sh allows. If you have deeper questions on this, I can hook you up with one of our project leads here who can chat about this more in-depth. Hope this helps though.
